website/siteConfig.js

```diff
@@ -10,50 +10,55 @@
 
 // List of projects/orgs using your project for the users page.
 
-const users = require('./data/users.json');
+// const users = require('./data/users.json');
+
+const alloyHighlighter = require('./alloyHighlighter');
 
 const siteConfig = {
-  title: ' ', // Title for your website. THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
+  title: 'alloy', // Title for your website. THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
   tagline: 'Language and visual platform to generate models for the financial services industry', //THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
-  cname: 'purealloy.finos.org',
-  url: 'https://purealloy.finos.org/',
+  cname: 'alloy.finos.org',
```

Comment:
@gs-bracej - qq. why this change? The current name is purealloy.finos.org - I've restored this value on `master` branch, please let me know if that is ok.

Reply:
Thanks! It was a snafu, caught up in the renaming from alloy -> k****s and back :)